Automattic/wp-calypso
# WooExpress: Moved eCommerce trial images to correct path

## Proposed Changes

*  Changed the path of the images used in the eCommerce Trial "My Plan" page
* Removed two unused images

## Testing Instructions

* Open the My Plan page of a trial site and check the images still show up correctly
![image](https://user-images.githubusercontent.com/3801502/218101158-b52eaa2a-f69e-4b7f-9701-28114bd643f3.png)
